url6da19b5a7a_krakrak_Adventure_Game_tgz-pJ8-adventure_MainJ8/src/items/SteelCompanionsSword.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package items;
+
+import NPC.Kill;
+import NPC.Killable;
+import NPC.Killable1;
+import NPC.Killable2;
+import NPC.Npc;
+import commands.Game;
+import commands.GameState;
+import commands.History;
+import java.util.List;
+import java.util.Random;
+
+/**
+ *
+ * @author Irakiza Krak
+ */
+public class SteelCompanionsSword extends AbstractWeapon {
+    
+    private Random rdm = new Random();
+    private Double removeNbLives;
+    private int nbLivesRemain;
+    
+    public SteelCompanionsSword(){
+        super("Steel Companion's Sword", "Large Steel two-handed sword. Typically a sword for an accomplishe Companion", "st s");
+    }
+
+    @Override
+    public GameState use(Game game) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 11. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.